examples/pytorch/graphsage/train_sampling_multi_gpu.py

```diff
@@ -118,13 +152,17 @@ def run(proc_id, n_gpus, args, devices, data):
 
         # Loop over the dataloader to sample the computation dependency graph as a list of
         # blocks.
-        for step, (input_nodes, seeds, blocks) in enumerate(dataloader):
+        for step, data in enumerate(dataloader):
             if proc_id == 0:
                 tic_step = time.time()
 
-            # Load the input features as well as output labels
-            batch_inputs, batch_labels = load_subtensor(train_nfeat, train_labels,
-                                                        seeds, input_nodes, dev_id)
+            if len(data) == 3:
```

Comment:
MultiGPUNodeDataLoader is a child of NodeDataLoader won't work. I could do the inverse:
```
if not isinstance(dataloader, MultiGPUNodeDataLoader):
```
But my thoughts were since we're unpacking `data`, we should depend on `data` to make the decision rather than on `dataloader` which isn't directly used in this statement. If `node_feat` and `node_label` weren't specified to `MultiGPUNodeDataLoader`, then it would also return only 3 arguments per-iteration and would work in that code path.